core/base/src/root-argparse.py

```diff
@@ -18,8 +18,9 @@ def get_argparse():
 	parser.add_argument('-h','-?', '--help', help='Show summary of options')
 	parser.add_argument('--version', help='Show the ROOT version')
 	parser.add_argument('--notebook', help='Execute ROOT notebook')
-	parser.add_argument('--web', help='Display graphics in a default web browser')
-	parser.add_argument('--web=<browser>', help='Display graphics in specified web browser')
+	parser.add_argument('--web', help='Use web-based display for graphics, browser, geometry')
+	parser.add_argument('--web=<kind>', help='Use specified kind for web-based display like chrome, firefox, qt5\nMore available options see in TROOT::SetWebDisplay() documentation')
```

Comment:
"Use the specified web-based display, such as .... For more options, see the documentation of TROOT::SetWebDisplay()"